todo/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="kr">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="todo.css" />
+    <script
+      src="https://kit.fontawesome.com/6d5791e2cf.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Web To Do Mate</title>
+  </head>
+  <body>
+    <header>
+      <h1>Web To Do Mate</h1>
+    </header>
+    <main>
+      <section class="weekly-section">
+        <article class="weekly-section__item">
+          <h3 class="weekly-section__item__day">월</h3>
+          <div class="weekly-section__item__to-do-num">
+            <img
+              class="weekly-section__item__to-do-num__heart"
+              src="./images/black-heart.png"
+              alt="검정 하트"
+            />
+            <span>6</span>
+          </div>
+          <span class="weekly-section__item__date">27</span>
```

Comment:
span은 시맨틱 태그가 아니라서 p 같은 거 사용해줘도 좋을 것 같아!
```suggestion
          <p class="weekly-section__item__date">27</p>
```